### ISSUE TITLE
FEAT: loading spinner

### ISSUE THREAD

**Issue by mcanepa** [beekeeper-studio/beekeeper-studio#1470]
IMHO a loading spinner would result in a better ui/ux than a static "loading" text while loading data

**Comment by rathboma** [comment]
@mcanepa agreed. Can you give me a few screen recordings of where this could be added? This is a great starter ticket for a new engineer, so if you can list all the places I can get them done :-)

**Comment by mcanepa** [comment]
[Captura del escritorio de 02-04-23 21:18:43.webm](https://user-images.githubusercontent.com/25961439/229387188-43b4cb78-ef8a-4810-a1e1-321783a3d04b.webm)
